includes/agentic-commerce/class-wc-stripe-agentic-commerce-inventory-tracker.php

```
@@ -0,0 +1,237 @@
+	 * @return string Stripe secret key.
+	 */
+	private function get_secret_key(): string {
+		$settings  = WC_Stripe_Helper::get_stripe_settings();
+		$test_mode = isset( $settings['testmode'] ) && 'yes' === $settings['testmode'];
+
+		if ( $test_mode ) {
+			return $settings['test_secret_key'] ?? '';
+		}
+
+		return $settings['secret_key'] ?? '';
+	}
```

PHPMD (2.15.0)

[error] The method get_secret_key is not named in camelCase. (undefined)

(CamelCaseMethodName)